src/wallet/coincontrol.h

```diff
@@ -16,7 +16,10 @@
 class CCoinControl
 {
 public:
+    //! Custom change destination, if not set an address is generated
     CTxDestination destChange;
+    //! Custom change type, ignored if destChange is set, defaults to g_change_type
```

Comment:
nit: the term "change type" seems not precise enough... I would have used `change address type`.